src/Styles/EndCurrentRoundModal.scss

```diff
@@ -0,0 +1,49 @@
+@import './_variables';
+@import './_mixins';
+
+.confirm-deletion-container {
+  position: fixed;
+  z-index: 2;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgba(0,0,0,0.4);
+  display: flex;
+  justify-content: space-between;
+}
+
+.confirm-deletion-modal {
+  display: flex;
```

Comment:
May be an opportunity for a mixin. The modals all have a lot of the same styling